test/jdk/javax/swing/text/CSSBorder/6796710/bug6796710.java

```diff
@@ -135,6 +137,41 @@ public void run() {
         System.out.println("The test bug6796710 passed.");
     }
 
+    public static boolean compareBufferedImages(BufferedImage bufferedImage0, BufferedImage bufferedImage1) {
+        int width = bufferedImage0.getWidth();
+        int height = bufferedImage0.getHeight();
+
+        if (width != bufferedImage1.getWidth() || height != bufferedImage1.getHeight()) {
+            return false;
+        }
+
+        for (int y = 0; y < height; y++) {
+            for (int x = 0; x < width; x++) {
+                Color bufCol0 = new Color(bufferedImage0.getRGB(x, y));
+                Color bufCol1 = new Color(bufferedImage1.getRGB(x, y));
+
+                int red1 = bufCol0.getRed();
+                int blue1 = bufCol0.getBlue();
+                int green1 = bufCol0.getGreen();
+
+                int red2 = bufCol1.getRed();
+                int blue2 = bufCol1.getBlue();
+                int green2 = bufCol1.getGreen();
+
+                if ((Math.abs(red1 - red2) > COLORTOLERANCE) ||
+                    (Math.abs(green1 - green2) > COLORTOLERANCE) ||
+                    (Math.abs(blue1 - blue2) > COLORTOLERANCE)) {
```

Comment:
Are binary operators usually wrapped to the next line or left on the same line?

Which style is the preferred one?

Reply:
I have seen the present style more common.